pkg/securitypolicy/securitypolicy.go

```diff
@@ -62,6 +62,19 @@ func NewContainerConfig(
 	}
 }
 
+// NewEnvVarRules creates an array of EnvRule's from environment variables array.
```

Comment:
slice* to be pedantic 😆

Reply:
I always go from `list` to `array` and never to `slice` 😄